migrations/2021-06-21-092747_create_users/up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE users (
+    id INTEGER PRIMARY KEY,
+    username VARCHAR(8) UNIQUE, -- For now, this is only zIDs, so we'll restrict to 8 characters.
+    display_name TEXT
```

Comment:
Don't particularly want to be using TEXT here (not the correct use-case).
Varchar with a sensible length should be okay, say varchar(30) or so